release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -764,6 +764,23 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
 
 {product-title} now supports the crun container runtime in Technology Preview. You can switch between the crun container runtime and the default container runtime as needed by using a `ContainerRuntimeConfig` custom resource (CR). For more information, see xref:../nodes/containers/nodes-containers-using.adoc#nodes-containers-runtimes[About the container engine and container runtime].
 
+[id="ocp-4-12-self-node-remediation-operator"]
+==== Self Node Remediation Operator enhancements
+
+{product-title} now supports control plane fencing by the Self Node Remediation Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#control-plane-fencing-self-node-remediation-operator_self-node-remediation-operator-remediate-nodes[Control Plane Fencing].
+
+[id="ocp-4-12-node-health-check-operator"]
+==== Node Health Check Operator enhancements
+
+{product-title} now supports control plane fencing on the Node Health Check Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#control-plane-fencing-node-health-check-operator_node-health-check-operator[Control Plane Fencing].
+
+The Node Health Check Operator now also includes a web console plugin for managing Node Health Checks. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#eco-node-health-check-operator-creating-node-health-check_node-health-check-operator[Creating a node health check].
+
+For installing or updating to the latest version of the Node Health Check Operator, use the `stable` subscription channel. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#installing-node-health-check-operator-using-cli_node-health-check-operator[Installing the Node Health Check Operator by using the CLI].
+
+[id="ocp-4-11-logging"]
```

Comment:
should this belong into this PR?

Reply:
> should this belong into this PR?

Thanks @slintes . No, lines 780/781 are not part of this PR. Maybe they got added my mistake. They point to 4.11 Logging, and are probably a placeholder for others to add that content. My content is immediately above this in lines 767-779 inclusive.